openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -355,6 +355,14 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     globalGeometryRules.setString(openstudio::GlobalGeometryRulesFields::RectangularSurfaceCoordinateSystem, "Relative");
     m_idfObjects.push_back(globalGeometryRules);
 
+    {
+      auto zoneAirContaminantBalances = model.getConcreteModelObjects<ZoneAirContaminantBalance>();
```

Comment:
I made an OSM in 1.10.5 and exported to IDF, the ZoneAirContaminantBalance object came through.